benchmarking/evaluations/scenarios/multiple_tokenization_goals.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+from api.models.enums import (
+    DiversifyType,
+    ScenarioAttribute,
+    TokenizationConstraintDirection,
+)
+from api.models.tokenization_constraint import TokenizationConstraint
+from benchmarking.evaluations.goals import DiversityGoal, WeightGoal
+from benchmarking.evaluations.interfaces import Scenario, Goal
+
+
+class MultipleTokenizationGoals(Scenario):
+    def __init__(self, value_of_gpa: int, value_of_race: int, value_of_age):
+        self.value_of_gpa = value_of_gpa
+        self.value_of_race = value_of_race
+        self.value_of_age = value_of_age
+
+    def name(self) -> str:
+        return "Scenario to run multiple tokenization goals"
```

Comment:
you can be more specific here, intent of this name was to be able to use it while generating figures